### ISSUE TITLE
Undocumented properties of organizationalBranding resource

### ISSUE THREAD

**Issue by MartinM85** [microsoftgraph/microsoft-graph-docs#21692]
When using Graph API SDK for .NET the properties below are defined in `Microsoft.Graph.Models.OrganizationalBrandingProperties` but missing in the documentation.

`customAccountResetCredentialsUrl`, `customCannotAccessYourAccountText`, `customCannotAccessYourAccountUrl`, `CustomCSS`, `CustomCSSRelativeUrl`, `CustomForgotMyPasswordText`, `CustomPrivacyAndCookiesText`, `CustomPrivacyAndCookiesUrl`, `CustomResetItNowText`, `CustomTermsOfUseText`, `CustomTermsOfUseUrl`, `Favicon`, `FaviconRelativeUrl`, `HeaderBackgroundColor`, `HeaderLogo`, `HeaderLogoRelativeUrl`, `LoginPageLayoutConfiguration`, `LoginPageTextVisibilitySettings`

Please add those properties into doc.

Second issue:

I'm using the SDK for the Graph API v1.0 but there is discrepancy between SDK and documentation for [loginPageLayoutConfiguration](https://learn.microsoft.com/en-us/graph/api/resources/loginpagelayoutconfiguration?view=graph-rest-beta) and [loginPageTextVisibilitySettings](https://learn.microsoft.com/en-us/graph/api/resources/loginpagetextvisibilitysettings?view=graph-rest-beta) resources. According to the documentation, these resources are not available in v1.0 but I can set them in SDK for v1.0.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: e34593e1-c45e-6a1b-7523-0cc8e4deb2a8
* Version Independent ID: 7a4cac36-e221-d6c2-4d9e-3aef74cbb400
* Content: [organizationalBranding resource type - Microsoft Graph v1.0](https://learn.microsoft.com/en-us/graph/api/resources/organizationalbranding?view=graph-rest-1.0)
* Content Source: [api-reference/v1.0/resources/organizationalbranding.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/v1.0/resources/organizationalbranding.md)
* Product: **identity-and-sign-in**
* Technology: **microsoft-graph**
* GitHub Login: @AlexanderMars
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by FaithOmbongi** [comment]
Thank you @MartinM85 for raising this issue. We're currently in the process of documenting these properties in the v1.0 docs and have this docs PR #21410 in progress. Kindly bear with us.

cc @quievey .

**Comment by MartinM85** [comment]
Important question: 

Can I set the properties mentioned in the question in v1.0? 

I'm asking because when trying to create a new organizational branding localization and setting properties with custom prefix

```
POST /organization/{organizationId}/branding/localizations

{
 ...
 "customAccountResetCredentialsUrl": "..."
 ...
}
```

It returns the error message:

> Expected property 'customAccountResetCredentialsUrl' is not present on resource of type 'OrganizationalBrandingLocalization'

It looks like that those properties are still not available in v1.0 and PR #21410 should be merge after v1.0 api will return those properties.

**Comment by FaithOmbongi** [comment]
Hi @MartinM85 - We don't consider APIs to be officially available for customers to use unless the corresponding Microsoft Graph REST API documentation is available. So yes, we can say the property isn't available in v1.0 today.
